src/Application/IRouteList.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (http://nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Nette\Application;
+
+use Nette;
+
+
+/**
+ * Routes collection.
+ */
+interface IRouteList extends \IteratorAggregate
```

Comment:
This is funny:

this works:
```php
interface A extends Traversable 
{}

class B implements IteratorAggregate, A
{
	function getIterator() {}
}
```

this triggers Fatal error: Class B must implement interface Traversable as part of either Iterator or IteratorAggregate:

```php
interface A extends Traversable 
{}

class B implements A, IteratorAggregate
{
	function getIterator() {}
}
```